src/extensions/step_handlers/script_handler/src/script_handler.cpp

```diff
@@ -225,17 +226,15 @@ ADUC_Result ScriptHandlerImpl::Download(const tagADUC_WorkflowData* workflowData
                 .retryTimeout = DO_RETRY_TIMEOUT_DEFAULT,
             };
 
-            result = ExtensionManager::Download(entity, workflowHandle, &downloadOptions, nullptr);
+            result = ExtensionManager::Download(&fileEntity, workflowHandle, &downloadOptions, nullptr);
+            ADUC_FileEntity_Uninit(&fileEntity);
         }
```

Comment:
Do we need this? We already call ADUC_FileEntity_Uninit at the end of the function.

Reply:
removed